### DISCUSSION TITLE
Busca por vaga de estágiario.

### DISCUSSION THREAD

**Discussion by David-Oliveira** [frontendbr/forum#977] [Discussão]
Como conseguir entrar no mercado de trabalho como estágiario? não consigo atender tudo que é exigido. Sou novo nesse mundo de programação, será que conseguirei algo somente no último semestre (estudo S. I) e estudando coisas especificas por fora da faculdade?

**Comment by laurindo** [comment]
Fala @David-Oliveira tenho tres amigos que estao se aventurando no mercado de tecnologia. Eles estão em busca de recolocação profisisonal e optaram por mudar de área e entrar nessa que está em alta.
Eles estão treinando diariamente pelo freecodecamp(https://www.freecodecamp.org/).

Não sei qual o seu gosto por linguagem ou em qual frente de desenvolvimento você curte mais. Por exemplo: front-end, back-end, banco de dados, analise de dados, QA, etc.

Mas se você deseja passar pelas etapas exigidas no mercado e que eu acho que está com um grande numero de oportunidades. Faz o freecodecamp. Ele cobre o front-end, back-end, devops, analise de dados, etc. São varias horas de curso totalmente gratuito.

E pode ter certeza que estudando isso você logo logo consegue sim uma vaga no mercado.

REssalto que existem tbm outras formas de estudos que a comunidade aqui ja indicou e possa indicar que tambem são muito boas. Mas essa é a minha dica que tenho pra você. Tenta todo dia aprender 1% de alguma coisa.

**Comment by David-Oliveira** [comment]
Obrigado pela atenção @laurindo!
Não conhecia o freecodecamp, vlw pela dica.
Gosto da área de front e também de bando de dados. Não sei qual caminho trilhar, mas vou me orientando pelas opniões expressas pela turma e o conhecimento adquirido na faculdade para escolher um segmneto.
Obrigado novamente.

**Comment by gionaufal** [comment]
Uma coisa importante a se desmistificar é que empresas normalmente pedem, nas vagas, muito mais do que vai ser necessário para o trabalho. É bem comum vermos vagas com várias linguagens e tecnologias como requisito, mas que na verdade a empresa só usa 1 ou 2. Então, se você souber o mínimo para a vaga, já vale a pena se inscrever, principalmente sendo estágio (que, teoricamente, não deveria exigir grande experiência em nada)

**Comment by David-Oliveira** [comment]
Obrigado @gionaufal !
Boa dica, realmente as empresas não deveriam exigir grande experiência...

**Comment by felipedotcom** [comment]
Além do FreeCodeCamp, tem o https://www.theodinproject.com que também é bem legal.
O que o @gionaufal disse é a pura verdade. Você tem que saber o mínimo e na minha opinião, esse mínimo é geralmente: html, css, javascritpt, sql. 
Tanto o freecodecamp como o odinproject, vão te dar uma base boa e ainda por cima você já monta seu portifólio com os projetos do curso.

**Comment by phenriqueleao** [comment]
Não conhecia o Odin Project, muito bom! Obrigado!

**Comment by gspaiva** [comment]
Fala @David-Oliveira, tranquilo? Eu sou estagiário mano, meu estágio é na área de desenvolvimento em geral. O que eu tenho certeza que me ajudou a entrar no mercado de trabalho foi eu ter começado a estudar sobre desenvolvimento web/desktop por conta própria quando eu tava no quarto período da faculdade, a partir desse momento eu consegui entrar em uma empresa júnior e ganhar mais experiência de mercado. Um conselho que eu te dou é : Faça projetos, faça cursos online, aprende a teoria e pratique bastante! O estágio será apenas uma consequência do seu esforço. Abraço

**Comment by David-Oliveira** [comment]
Obrigado pela atenção @gspaiva !
Estou começando os estudos além da faculdade, mas fico pensando  em pegar experiência na área o mais breve possível. E claro, não posso ser ingrato por ter um emprego, mas é muito triste trabalhar em algo fora do que realmente deseja!
Bem vamos aprimorando, uma hora a oportunidade aparece e temos que estar preparado.
